contributing.md

```diff
@@ -1,18 +1,65 @@
 
-Contributing
-============
+#Contributing#
 
-Issues:
--------
+##Issues:##
 
- * If possible link to example app which replicates the problem
- * If it's a security issue, please email us rather than filing an issue
+ * If possible link to example app which replicates the problem.
+ * If it's a security issue, please email us rather than filing an issue.
 
-Pull Requests:
---------------
+###Labelling system:
 
- * Pull requests are always welcome
- * We prioritize PRs over issues
+OK GROW! follows a issue labelling system which is described below.
+
+####Primary label:
+
+All issues are to have a primary label applied.
+
+* **bug**
+  * An issue that causes a problem in the app.
+* **question** 
+  * Asking for clarification or guidance regarding to the specific package.
+* **enhancement** 
+  * Feature request for the specific package.
+
+####Secondary label:
+
+A secondary label will be applied once a member of OK GROW! has reviewed the issue.
+
+* **fixed**
+  * Issue has been resolved.
+* **wont-fix**
+  * OK GROW! will not be implementing a fix for the specific issue raised.
+* **duplicate**
+  * Issue has been previously raised, this Issue will be closed and link to the original issue raised.
+* **in-progress**
+  * Issue is currently being worked on to resolve.
+  * *NOTE:* This label is to be removed once completed.
+* **under-consideration**
+  * Enhancement is currently being considered by members of OK GROW!.
+  * *NOTE:* This label is to be removed once the enhancement has been accepted or rejected.
```

Comment:
@pauldowman Sounds good, I think we should make a point of stating that when a feature-request is under-consideration it doesn't mean that we will accept a PR if one is produced by someone. It has the label under-consideration as we are discussing if we want to support the enhancement and the maintenance cost for adding it. Robert suggested we make that clear to people wanting to contribute to the package.